traffic_ops/traffic_ops_golang/deliveryservice/keys.go

```diff
@@ -150,25 +201,23 @@ func GetSSLKeysByHostName(w http.ResponseWriter, r *http.Request) {
 	// lookup the cdnID
 	cdnID, ok, err := getCDNIDByDomainname(domainName, inf.Tx.Tx)
 	if err != nil {
-		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New("getting cdn id by domain name: "+err.Error()))
-		return
+		return "", false, errors.New("getting cdn id by domain name: " + err.Error())
 	}
 	if !ok {
-		api.WriteRespAlert(w, r, tc.InfoLevel, " - a cdn does not exist for the domain: "+domainName+" parsed from hostname: "+hostName)
-		return
+		log.Warnf(" - a cdn does not exist for the domain: " + domainName + " parsed from hostname: " + hostName)
```

Comment:
you don't need these dashes, btw. `"a CDN does not exist for ...` works just fine. Also, since you're already using `log.Warnf`, you should probably use format strings to interpolate those variables like `log.Warnf("domain: %s parsed from hostname: %s", domainName, hostName)`. If you like concatenating strings with `+` better (some do), then I'd advise you use [`log.Warnln`](https://godoc.org/github.com/apache/trafficcontrol/lib/go-log#Warnln) instead.

But also, this looks more like an error, and in fact results in an error response to the client. So probably use [`log.Errorln`](https://godoc.org/github.com/apache/trafficcontrol/lib/go-log#Errorln)/[`log.Errorf`](https://godoc.org/github.com/apache/trafficcontrol/lib/go-log#Errorf) instead.

Reply:
done